BuildingBlocks/src/BuildingBlocks.Infrastructure/Persistence/BlobStorage/GoogleCloudStorage/GoogleCloudStorage.cs

```diff
@@ -88,7 +93,7 @@ private void EliminateNotFound(Exception ex, string blobId)
 
     public async Task SaveAsync()
     {
-        await UploadChangedBlobs();
+        await _logger.TraceTime(async () => UploadChangedBlobs(), nameof(UploadChangedBlobs));
         await DeleteRemovedBlobs();
```

Comment:
There is still no time logged for this call... Same for AzureStorageAccount

Reply:
I just checked my local files - it's definitely there, it also seems to be committed and pushed. Can you please check again?

Reply:
https://github.com/nmshd/backbone/blob/c2289f60f52249335a75c405ac723cfba71176ea/BuildingBlocks/src/BuildingBlocks.Infrastructure/Persistence/BlobStorage/GoogleCloudStorage/GoogleCloudStorage.cs#L97

https://github.com/nmshd/backbone/blob/c2289f60f52249335a75c405ac723cfba71176ea/BuildingBlocks/src/BuildingBlocks.Infrastructure/Persistence/BlobStorage/AzureStorageAccount/AzureStorageAccount.cs#L87

This is the current state on your branch.

Reply:
Ok, I thought it refers to `UploadChangedBlobs()`. All done now.